### PR TITLE
tickets/DM-33449

### PR DESCRIPTION
**Pull request by mareuter**
Adding TTS text to control system upgrade documentation.

* Add TTS to Pre-Deployment Activites section.
* Add TTS to Deployment Activities section.
* Add comment about NTS retirement.
* Adjust documentation for other changes.

### CODE REVIEW

File: Operational-Procedures/Control-System-Upgrade/Deployment-Activities/index.rst [hunk 1]
```diff
@@ -36,30 +34,32 @@ If changes are necessary to these scripts from work described in the previous se
 
 #. With all the systems OFFLINE, you can log out of your Nubaldo instance as we will clean them up soon.
 #. Once all systems are in OFFLINE, still running CSCs/systems and OSPL daemons need to be cleaned up.
-    #. Get number of currently running daemons/CSCs still in OFFLINE from main OSPL daemon:
-        * *docker exec ospl-daemon grep "federations" durability.log*
+    #. Get number of currently running daemons/CSCs still in OFFLINE from main OSPL daemon: (:ref:`TTS <Deployment-Activities-TTS-Federation-Check>`)
+        * *docker exec ospl-daemon grep "federations" durability.log*
```
Comment (wvreeven) [inline Operational-Procedures/Control-System-Upgrade/Deployment-Activities/index.rst:38]: Please revert this change and remove the trailing space.

**Review by wvreeven — APPROVED** [review]
Looks good to me. Please have a look at my one tiny comment.

File: Operational-Procedures/Control-System-Upgrade/Deployment-Activities/index.rst (hunk 1 above)
Comment (mareuter) [reply Operational-Procedures/Control-System-Upgrade/Deployment-Activities/index.rst]: Fixed.
